### PR TITLE
Update plugin doc

### PR DESCRIPTION
**Pull request by jadb**
Refs: https://github.com/cakephp/app/pull/206

Here's an attempt to update the documentation. I haven't changed anything but the english version (I could probably check the french one once you guys confirm the final english version for this PR).

If anyone thinks there should be more instructions/documentation, please suggest (write it, I'll paste it, hehe) and/or guide me.

### CODE REVIEW

File: en/plugins.rst [hunk 1]
```diff
@@ -27,14 +27,19 @@ would do the following::
...
     php composer.phar require cakephp/debug_kit
 
-This would install the latest version of DebugKit and update your
+This would install the latest version of DebugKit in ``plugins`` and update your
 ``composer.json``, ``composer.lock`` file, and update your autoloader. If
 the plugin you want to install is not available on packagist.org, you can clone
-or copy the plugin code into your ``plugins`` directory. Assuming you want to install
-a plugin named 'ContactManager', you should have a folder in ``plugins``
+or copy the plugin code into your ``src/Plugin`` directory. Assuming you want to install
+a plugin named 'ContactManager', you should have a folder in ``src/Plugin``
```
Comment (markstory) [inline en/plugins.rst]: You wouldn't be installing, you would be creating.

File: en/plugins.rst [hunk 2]
```diff
@@ -237,14 +242,14 @@ Plugin Controllers
...
 
 Controllers for our ContactManager plugin will be stored in
-``plugins/ContactManager/src/Controller/``. Since the main thing we'll
+``src/Plugin/ContactManager/src/Controller/``. Since the main thing we'll
 be doing is managing contacts, we'll need a ContactsController for
 this plugin.
 
 So, we place our new ContactsController in
-``plugins/ContactManager/src/Controller`` and it looks like so::
+``src/Plugin/ContactManager/src/Controller`` and it looks like so::
```
Comment (markstory) [inline en/plugins.rst]: I thought it was `/Plugin`? not `src/Plugin`?

File: en/plugins.rst [hunk 3]
```diff
@@ -81,8 +86,8 @@ should be like::
 
     "psr-4": {
         (...)
-        "AcmeCorp\\Users\\": "./plugins/AcmeCorp/Users/src",
-        "AcmeCorp\\Users\\Test\\": "./plugins/AcmeCorp/Users/tests"
+        "AcmeCorp\\Users\\": "./src/Plugin/AcmeCorp/Users/src",
```
Comment (markstory) [inline en/plugins.rst]: Isn't it a bit odd to have two `src` directories in the path? Using `/Plugin` would only require one.

Comment (dereuromark) [inline en/plugins.rst]: I don't think it's too bad. At least it is consistent.

Comment (markstory) [inline en/plugins.rst]: Ok

**Comment by lorenzo** [comment]
This won't be required after the changes to the plugin installer. Thanks anyway @jadb

**Pull request closed without merging**